src/FSharpLint.Console/Program.fs

```diff
@@ -22,12 +22,14 @@ type private FileType =
 type private ToolArgs =
     | [<AltCommandLine("-f")>] Format of OutputFormat
     | [<CliPrefix(CliPrefix.None)>] Lint of ParseResults<LintArgs>
+    | Version 
 with
     interface IArgParserTemplate with
         member this.Usage =
             match this with
             | Format _ -> "Output format of the linter."
             | Lint _ -> "Runs FSharpLint against a file or a collection of files."
+            | Version -> "Prints current version"
```

Comment:
@Thorium all other strings before this one contain a dot at the end, so let's be consistent here